hoot-core-test/src/test/cpp/hoot/core/algorithms/MaximalNearestSublineTest.cpp

```diff
@@ -64,9 +64,9 @@ class MaximalNearestSublineTest : public HootTestFixture
 
 public:
 
-  virtual void setUp()
+  MaximalNearestSublineTest()
   {
-    HootTestFixture::setUp();
+    setResetType(ResetAll);
```

Comment:
`ResetAll` is set in order to remove the calls to `TestUtils::resetEnvironment` below.